src/app/shared/components/template/components/audio/audio.component.scss

```diff
@@ -1,4 +1,7 @@
 @use "/src/theme/mixins";
+
+$ion-btn-control-bg: var(--ion-btn-control-bg, var(--ion-color-primary-500));
```

Comment:
nit(non-blocking) - Something I've started noticing is a lot of the previous naming conventions seem to confuse what are variables coming from or overriding those provided by ionic (e.g. `--ion-color-primary`) vs ones we have created and are using and have just given a `--ion` prefix to for whatever reason. 

I really think it would be good to decide a way to start disambiguating (maybe add as a future action item on the existing checklist?). Really we should only use `--ion` for things designed to interact with and override ionic components. Anything else should either just be simple names (ionic components themselves mostly just refer to `$background` or similar before being overwritten by global variable), or possibly prefixed in a standard way (e.g. `--theme-button-control-background`)